docs/recipes/isolating-mongodb-integration-tests.md

```diff
@@ -0,0 +1,86 @@
+# Setting up Ava for Isolated MongoDB Integration Tests
+
+This recipe outlines how to run disposable MongoDB databases in your Ava tests, isolated between tests.
+
+## Install MongoDB in-memory Server (MongoMem)
+In the root directory of your application, run:
+
+`npm install mongomem --save-dev`
+
+## Using MongoMem
+In your test file, import the module and run the server. 
+*Make sure to run the server at the start of your file, outside of any test cases.*
+
+```javascript
+import { MongoDBServer } from 'mongomem';
+let hasStarted = MongoDBServer.start();
+
+// ES6 promise syntax
+test('some feature - es6', t => {
+	// wait for server to be ready
+	hasStarted.then(() => {
+		MongoDB.getConnectionString().then(connectionString => {
+			// run logic with access to connectionString
+		});
+	});
+});
+
+// async/await syntax
+test('some feature - async/await', async t => {
+	await hasStarted;
+	let connectionString = await MongoDBServer.getConnectionString();
```

Comment:
This should probably use `const`.

What should users do next, once they have the `connectionString`?